corehq/form_processor/interfaces.py

```diff
@@ -146,6 +145,7 @@ def get_case_xform_ids_from_couch(case_id):
 
     @staticmethod
     @to_generic
+    @unit_testing_only
     def post_xform(instance_xml, attachments=None, process=None, domain='test-domain'):
```

Comment:
Why put unit testing functions in a module with production code? Would it make sense to create a subclass of `FormProcessorInterface` with this method in `corehq.form_processor.tests`?